Source/WebCore/css/mathml.css

```diff
@@ -44,6 +44,7 @@
 /* Universal rules */
 * {
     writing-mode: horizontal-tb !important;
+    display: block;
```

Comment:
Can you please add a FIXME comment referring to bug 278533?

Reply:
Done in https://github.com/WebKit/WebKit/commit/b34d57188d64ad705feb68336e9367605315401e.